### ISSUE TITLE
Lutris fix

### ISSUE THREAD

**Issue by byrongibson** [NixOS/nixpkgs#173712]
### Describe the bug

Lutris needs [this fix](https://github.com/lutris/lutris/commit/072e72a4aefd91101b79dd05d8ce9f100a4b6b0c) applied.  (Edit: or maybe better to just update to latest Lutris version 9.10 which includes the fix?).  Without it, programs can sometimes hang while launching.  

This is not a NixOS-specific bug, see discussions of it on other distro's here:

1.  https://www.reddit.com/r/Lutris/comments/t6fbxl/cant_launch_any_titles_with_lutris_python_error/
2.  https://forums.lutris.net/t/all-games-hang-on-launching/14754/4
3.  https://forums.lutris.net/t/lutris-doesnt-run-anything/15179/4

### Steps To Reproduce
1. Install and open Lutris
2. Select a program and click the Play button
3. The "Play" button is replaced with "Launching" button, but the program hangs here, no program opens, and the "Launching" button remains indefinitely.

### Expected behavior

Click "Play" button and the program opens.

### Screenshots

See links in Description section above.

### Additional context

[A simple 1-line fix.](https://github.com/lutris/lutris/commit/072e72a4aefd91101b79dd05d8ce9f100a4b6b0c)

### Notify maintainers

@chiiruno

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"

 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.26, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.16`
 - channels(root): `"nixos-21.11.336386.bacbfd713b4"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

**Comment by Ice-Cube69** [comment]
This fix has been updated in Lutris 9.10, So I believe updating lutris in the repo is a better solution.

**Comment by Ice-Cube69** [comment]
I was trying to follow this comment: https://github.com/lutris/lutris/issues/4175#issuecomment-1082578219
but I was not able to write in /nix/store, so if someone can provide me a suitable solution, that would be great as well!

**Comment by Ice-Cube69** [comment]
@rapenne-s Can you take a look?

**Comment by rapenne-s** [comment]
I've been using Lutris on NixOS unstable for a while, it works fine.

I suppose it would be better to backport the lutris version from unstable to 21.11 if possible, rather than adding the fix.

**Comment by Ice-Cube69** [comment]
@rapenne-s People at the matrix chat were also saying the same, so should I make a PR or what?

**Comment by rapenne-s** [comment]
@Ice-Cube69 yes, making a PR will make the things move if it's what you want. However 22.05 is out now, lutris should be fixed in it I suppose.

**Comment by byrongibson** [comment]
Fixed in 22.05.
